src/octopeer-github/test/content/ContentControllerTest.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * Created by Mitchell on 25-5-2016.
+ * Behaviour unit tests for the ContentController class.
+ */
+
+describe("The ContentController", function() {
+    let spy: jasmine.Spy;
+    let spyAddedListeners: jasmine.Spy;
+    let spyCheckListeners: jasmine.Spy;
+
+    let testContentController: ContentController;
+
+    const propertyListOne = [
+        ClickElementEventBinding,
+        KeystrokeElementEventBinding,
+        MouseEnterElementEventBinding,
+        MouseLeaveElementEventBinding,
+        ScrollIntoViewElementEventBinding,
+        ScrollOutOfViewElementEventBinding,
+    ];
+
+    const propertyListTwo = [
```

Comment:
I would go for some more descriptive names here, like `elementEventBindingList` and `elementSelectionBehaviourList`.